src/artifacts.js

```diff
@@ -356,6 +356,9 @@ var replyWithArtifact = async function(taskId, runId, name, req, res) {
       if (!region) {
         debug('artifact from CDN for ip: %s', req.headers['x-forwarded-for']);
         url = this.publicBucket.createGetUrl(prefix);
+      } else if (req.headers['x-taskcluster-skip-cache'] === 'true') {
```

Comment:
should we `.toLowerCase()` the header value before comparison?  Maybe we should also support `1|0`?